skins/laika/templates/page_layouts/base.html.twig

```diff
@@ -3,6 +3,8 @@
 {% set title = site_metadata.page_titles[page_id] %}
 {% set content = web_content("pages/#{page_id}") %}
 
+{% block title %}{$ page_title( page_id ) $}{% endblock %}
```

Comment:
```javascript
{% block title %}{$ page_title( page_id ) $}{% endblock %}
```

In this line the translation key is `page_id` because the file is `base.html.twig`?

Reply:
Base is used for the static pages, and page_id is the key that the individual page translation is stored under in the content repo.

If you look at the change I made in the `WebTemplatingFactory.php` file you can see I defined a new function to pull out the title based on that key.